File: tests/transpiler/x/fortran/basic_compare.f90
program main
  implicit none
  integer :: a = 10 - 3
  integer :: b = 2 + 2

  print '(I0)', a
  print '(A)', trim(merge('true  ','false ',a == 7))
  print '(A)', trim(merge('true  ','false ',b < 5))
end program main

File: tests/transpiler/x/pas/basic_compare.pas
{$mode objfpc}
program Main;
var
  a: integer;
  b: integer;
begin
  a := 10 - 3;
  b := 2 + 2;
  writeln(a);
  writeln(ord(a = 7));
  writeln(ord(b < 5));
end.
